frontend/model/contracts/group.js

```diff
@@ -830,7 +836,11 @@ sbp('chelonia/defineContract', {
           })
         }
 
-        sbp('gi.contracts/group/emitNotificationsAfterSyncing', contractID, notifications)
+        const { loggedIn } = sbp('state/vuex/state')
```

Comment:
Since `'gi.contracts/group/emitNotifications'` does nothing significant anymore, it can be removed, and instead of calling it like this, `notifications` array can be deleted.

Then the loop above can be rewritten, something like this:

```js
if (isActionNewerThanUserJoinedDate(height, myProfile)) {
  for (const proposalId of data.proposalIds) {
    sbp('gi.notifications/emit', 'PROPOSAL_EXPIRING', {
      // ...
    })
  }
}
```